### DISCUSSION TITLE
Welcome to VoodooHDA-OC Discussions!

### DISCUSSION THREAD

**Discussion by chris1111** [chris1111/VoodooHDA-OC#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jimmy26869** [comment]
Hello, I am pretty new to Hackintosh and especially to Opencore (having some experience with Clover) but I thought I would add something to the VoodooHDA discussion based on my experience. After getting Monterey installed using Opencore I was unable to get any sound working. I tried everything I could find on the web including every layout id imaginable in both the boot arguments and under audio devices in my config list. Finally, I struck upon instructions to install chris1111's VoodooHDA but after installing it in the Preference Pane, deleting it from the config list following these instructions:[ ](https://www.youtube.com/watch?v=Cgd0nkjKUyE) Anyway, magically the sound devices were now visible and working. However, I usually need to boot at least twice in order for the sound devices to load. The first time I boot, there are no devices, often the second time there is digital HDMI only then the third boot gives me the headphone and onboard analog speakers. Not sure if this will be useful to anyone else but thought I would share anyway.

**Reply by Jimmy26869** [reply]
https://www.youtube.com/watch?v=Cgd0nkjKUyE
These are the instructions to install VoodooHDA I used
